components/http-accept.js

```diff
@@ -19,35 +19,43 @@ exports.getComponent = promiseComponent({
             description: "HTTP 202 Accepted request/response pair {req, res}",
             datatype: 'object'
         }
-    },
+    }, promiseOutput.outPorts),
     inPorts: {
         limit: {
             description: "Request Content-Length limit",
             datatype: 'int',
-            ondata: basenode.assign('limit')
+            ondata: function(limit) {
+                this.nodeInstance.limit = limit;
```

Comment:
Yes, with noflo-access branch merged in, this would be assigned to the component facade